test/test_projects_api.py

```diff
@@ -52,8 +52,8 @@ def test_retrieve_projects(self):
             project = projects[0]
 
             self.assertEqual(project.production, False)
-            self.assertEqual(project.average_price_per_tonne_cents_usd, 0)
-            self.assertEqual(project.remaining_mass_g, 0)
+            self.assertGreater(project.average_price_per_tonne_cents_usd, 0)
+            self.assertGreater(project.remaining_mass_g, 0)
```

Comment:
@biglovisa I had to relax this test because we added records to this project on the sandbox.
Generally, I wouldn't want those tests to be too locked in to a specific value.